project/Dependencies.scala

```diff
@@ -70,11 +70,11 @@ object Dependencies extends DependencyBuilders {
   val LoggingDependencies: Seq[ModuleID] =
     Seq(
       "com.typesafe.scala-logging" %% "scala-logging"           % "3.9.4",
-      "net.logstash.logback"       % "logstash-logback-encoder" % "7.0.1"
+      "net.logstash.logback"       % "logstash-logback-encoder" % "7.1.1"
```

Comment:
hum... no sure when/how I did that... let me know if you want met to put it back